daisy_workflows/build-publish/rhui/rpms/rh-client-config.spec

```diff
@@ -77,7 +79,8 @@ rm -rf $RPM_BUILD_ROOT
 %if 0%{?_gpg_keys:1}
     %config  /etc/pki/rpm-gpg/RPM-GPG-KEY-*
 %endif
+%attr(755,root,root) /etc/cron.daily/google-rhui-client-package-update
```

Comment:
Now that I found the [Fedora docs](https://fedoraproject.org/wiki/Packaging:CronFiles#Cron_Job_Files_Packaging) on this, it should be a config file with the noreplace annotation
%config(noreplace)

Reply:
done